src/client/components/StaffViewRoute/index.js

```diff
@@ -13,10 +12,7 @@ function StaffViewRoute(props) {
   const { pageComponent: Component, pageProps, ...routeProps } = props;
 
   let routeChild = <div>Loading...</div>;
-  if (isProdEnv) {
-    // Staff view should currently not load on production at all,
-    routeChild = <Redirect to={routes.retroCertsAuth} push />;
-  } else if (
+  if (
```

Comment:
want to add a comment about staff view being restricted to EDD IP addresses in app.js?